.goreleaser.yaml

```diff
@@ -10,6 +10,7 @@ before:
 builds:
   - env:
       - CGO_ENABLED=0
+    ldflags: "-s -w -X main.version={{ .Version }}"
```

Comment:
```suggestion
    ldflags: "-s -w -X main.Version={{ .Version }}"
```

Reply:
ahh yee...

Reply:
later on, we should make `version` CLI command as well